packages/carbon-graphs/src/js/controls/Graph/GraphConfig.js

```diff
@@ -338,15 +231,136 @@ class GraphConfig extends BaseConfig {
     return this;
   }
 
+  // TODO: cloning may bloat up the the graph config object by adding additional properties.
+  // This may also be an unecessary or redundant step as processInput adds in default values
+  // if any are missing. A spike needs to be done to confirm if this step is unecessary and
+  // can be removed.
+
   /**
      * Clones the input JSON into the config object
      *
      * @returns {GraphConfig} instance object
      */
-  clone() {
+  cloneInput() {
     this.config = utils.deepClone(this.input);
     return this;
   }
+
+  // TODO: this method needs to be added to the baseConfig class as part of the class template
+
+  /**
+  * Processes the input from the JSON and updates the config object.
+  * d3 domain and ranges are stored based on the upper and lower x, y and y2 limits.
+  *
+  * @private
+  * @param {object} input - Input JSON provided by the consumer
+  * @param {object} config - config object used by the graph
+  * @param {string} type - input type
```

Comment:
These params can probably be removed since processInput no longer takes these.

Reply:
Removed here: https://github.com/cerner/terra-graphs/pull/285/commits/176c658808cb2290e8ac28cb97d92bbaf0188500